src/btree/bt_sync.c

```diff
@@ -194,6 +194,7 @@ __sync_ref_evict_or_mark_deleted(WT_SESSION_IMPL *session, WT_REF *ref)
 
     /* Evict the in-memory obsolete page */
     WT_IGNORE_RET_BOOL(__wt_page_evict_urgent(session, ref));
+    WT_STAT_CONN_INCR(session, hs_gc_pages_removed);
```

Comment:
The `removed` stat here doesn't provide the actual pages that are removed.  The removed case is in the above if check. Move the stat there.

Reply:
Thanks!  Done.